.github/workflows/release-embedding-sdk.yml

```diff
@@ -186,15 +186,15 @@ jobs:
         run: |
           bash ./bin/embedding-sdk/release_utils.bash update_package_json_template ${{ inputs.sdk_version }}
 
-      - name: Retrieve SDK changelog
+      - name: Retrieve SDK changelog diff
         uses: actions/download-artifact@v4
         with:
-          name: sdk-changelog
+          name: sdk-changelog-diff
 
-      # We need to do this because it's impossible to download an artifact to an existing location
-      - name: Move over the downloaded changelog
+      - name: Update changelog
         run: |
-          mv CHANGELOG.md ./enterprise/frontend/src/embedding-sdk/CHANGELOG.md
+          cat changelog-diff enterprise/frontend/src/embedding-sdk/CHANGELOG.md > new-changelog
```

Comment:
Found on some Stackoverflow thread that cat append multiple file into a single output. That's neat!

But there don't seem to be a way to do this in one line, so I need to create a temp file before replacing it with the current changelog.

This is tested that it worked since I've generated the changelog with [this commit](https://github.com/metabase/metabase/pull/43729/commits/0e77a68e1b1cc1ca8d962c73b8ca774941ef582a) ([proof](https://github.com/metabase/metabase/actions/runs/9403410194/job/25900153215#step:2:3)), which don't have the same content as master's changelog.

You can see that the changelog [was created correctly.](https://github.com/metabase/metabase/pull/43740/files#diff-d36366ea0c0db75f1f528cae974feec799d37b91ce54c93a27881c85fb96b8c1)